install-storageos/060-access-ui.md

```diff
@@ -0,0 +1,10 @@
+StorageOS serves the GUI on the port 5705 of each DaemonSet Pod.
```

Comment:
Please mention here that if customers see issues they should disable ad-block and privacy checkers on the new window.